src/mongoose_lib.erl

```diff
@@ -169,6 +171,9 @@ does_local_user_exist(HostType, To, groupchat) ->
 does_local_user_exist(HostType, To, _) ->
     ejabberd_auth:does_user_exist(HostType, To, stored).
 
+%% ------------------------------------------------------------------
+%% parallel map
+%% ------------------------------------------------------------------
```

Comment:
This should actually go a few lines below, here there's still the `is_to_room/1` function xD